frontend/src/components/QuizRedo.js

```diff
@@ -1,7 +1,35 @@
 import React, { Component } from 'react';
-import { Button, Modal, ModalHeader, ModalBody, ModalFooter } from 'reactstrap';
+import {
+  Button,
+  Modal,
+  ModalHeader,
+  ModalBody,
+  ModalFooter,
+  Progress
+} from 'reactstrap';
 
 class QuizRedo extends Component {
+  renderProgress = () => {
+    let correct = (this.props.numCorrectQ / this.props.numTotalQ) * 100;
+    let incorrect =
+      ((this.props.numTotalQ - this.props.numCorrectQ) / this.props.numTotalQ) *
+      100;
+    return (
+      <div
+        style={{
```

Comment:
not sure if necessary, but should this be a class or is it okay to be an inline style? @XuanLi-Leong

Reply:
Yeaaah if we could throw it into a class in quizpage.scss that'd be 💯